repository/mock_repo.go

```diff
@@ -82,6 +82,11 @@ func (r *mockRepoForTest) RmConfigs(keyPrefix string) error {
 	return nil
 }
 
+func (r *mockRepoForTest) ListRemotes() ([]string, error) {
+	testRemotes := make([]string, 2)
```

Comment:
A mock (see https://stackoverflow.com/questions/214092/what-is-a-mock-and-when-should-you-use-it) should return correct values, and an array of two empty string is not a correct return value for `ListRemotes`